test/gpu/native/localeName.prediff

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+hn=$(uname -n)
+sed "s/PREDIFF_HOSTNAME/$hn/" < localeName.good.in > localeName.good
```

Comment:
Let's simplify this:

- `prediff` will have its `$2` have the executable output (`foo.exec.out.tmp`)
- So 
```
sed "s/$hn/PREDIFF_HOSTNAME/" $2 > $2.predifftemp  # Note the reversed find/replace
mv $2.predifftemp $2
```
is how we typically do this.
- It would mean your `good.in` is actually just your `good` file
- It would also mean you don't need `cleanfiles`

We *do* generate `good` files in `prediff`s in some cases, but there's no complication here that requires that IMO. And we better stick with the common practice. `find test -name "*prediff" | xargs -L1 grep -L mv` shows prediffs that `mv`s things. Remove `-L` to see the `mv` commands that look like what I described above.

Reply:
Got it! That should be fixed now.